clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -486,6 +486,11 @@ public enum Command {
       "Dump the real-time ingestion state for a certain store version in a certain storage node",
       new Arg[] { SERVER_URL, STORE, VERSION }, new Arg[] { PARTITION }
   ),
+  DUMP_TOPIC_PARTITION_INGESTION_CONTEXT(
+      "dump-topic-partition-ingestion-context",
+      "Dump the topic partition ingestion context belong to a certain store version in a certain storage node",
+      new Arg[] { SERVER_URL, STORE, VERSION, KAFKA_TOPIC_NAME, KAFKA_TOPIC_PARTITION }
```

Comment:
Hmm is it redundant parameter? I mean store_version is the Kafka Topic name?

Reply:
No,  kafka topic name could be real-time topic, it is possible it belongs to 2 store versions. Here we use version topic and topic partiton to locate the exact consumer.